src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { MuiForm } from '.';
+
+const meta = {
+	title: 'Other/MuiForm',
+	component: MuiForm,
+	tags: ['autodocs'],
+} satisfies Meta<typeof MuiForm>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
```

Comment:
Your submit button comes out as a small textless button (same with the secondary button if you try to add it): 
![image](https://github.com/balena-io-modules/ui-shared-components/assets/22177187/da995ef7-c4db-4ee1-b80d-f158de8b12a5)

Reply:
thanks for noticing, just forgot to add a default value after removing the not needed property submitButtonText